cg-spec/editors_draft.html

```diff
@@ -415,12 +414,124 @@ <h2>Concepts and Abstract Syntax</h2>
 
     <p>Again, this definition is an extension of the notion of <a>RDF dataset</a>, hence it follows that any <a>RDF dataset</a> is also an <a>RDF-star dataset</a>.</p>
 
+    <section class="informative" id="quoted-triples">
+      <h2>Quoted and Asserted Triples</h2>
+
+      <p>
+        A triple in RDF is a tuple of three components: subject, predicate and object. These are
+        collectively known as <dfn data-cite="RDF11-CONCEPTS#section-rdf-graph">RDF terms</dfn>.
+      </p>
+      <p>
+        RDF-star introduces <em>quoted triple</em>, which a new kind of RDF term.
+        A <em>quoted triple</em> is a triple used as the subject or object of another triple. Quoted
+        triples can also be called "embedded triples".
+      </p>
+
+      <p>
+        in RDF 1.1, an <em>asserted triple</em> is a element of the set of triples that
+        make up an <a>RDF graph</a>. RDF-star does not change this except that an RDF-star
+        triple can contain quoted triples. A triple can be used as an <em>asserted
+        triple</em> and as a <em>quoted triple</em> in the same graph.
+      </p>
+
+      <p>
+        A quoted triple is written in Turtle-star and related syntaxes using delimiters `<<` and `>>`.
+      </p>
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            << :a :name "Alice" >>
+        -->
+      </pre>  
+      <p>
+        The definition of quoted triple is recursive. A quoted triple can itself have a
+        subject or object component which is another quoted triple. Cycles are not allowed
+        and can not be written in any RDF syntax. In this next example, there is a quoted
```

Comment:
It may be short-sighted to say that it can't be written in any RDF syntax, if this becomes RDF syntax. Perhaps "any RDF syntax not supporting _embedded triples_", or similar wording.

Reply:
Looking at normative section, the definition notes that cycles aren't possible. The text here can be simplified.